code/headers/test_usart.hpp

```diff
@@ -47,5 +48,17 @@ namespace r2d2 {
         ///@return unsigned int always 1
```

Comment:
this doesn't always return 1 anymore

Reply:
edited in  https://github.com/R2D2-2019/USART_library/pull/12/commits/85fdce96b4074228fcd84ff4c1db1e03da94d3f2